assets/js/base/context/cart-checkout/validation/index.js

```diff
@@ -98,6 +165,10 @@ export const ValidationContextProvider = ( { children } ) => {
 		clearValidationError,
 		clearAllValidationErrors,
 		getValidationErrorId,
+		hideValidationError,
+		showValidationError,
+		showAllValidationErrors,
+		areThereValidationErrors,
```

Comment:
Maybe rename this to `hasValidationErrors`?

Reply:
Sure thing! Renamed in a21df92d6cd5ee85b2c7b66298205e479f1ad499.